src/sentry/static/sentry/app/views/groupDetails/shared/actions.jsx

```diff
@@ -109,7 +109,12 @@ class DeleteActions extends React.Component {
 const GroupDetailsActions = createReactClass({
```

Comment:
Since you removed the mixes we could migrate this over to a ES6 class too :)

Reply:
👍 Will try to get to this soon